_includes/saved_searches.md

```diff
@@ -0,0 +1,62 @@
+## Save and Manage Searches
+
+Because narrowing down a search in the Application Status page or Traces Browser can be time consuming, you can
+* Save a search. Saved searches are available to all members of your organization.
+* Access and delete your saved searches
+* Access saved searches from everyone in your organization.
+* Set a saved search as the default search.
+
+### Save and Access a Search
+
+After you've performed a search on the Application Status page or Traces Browser, you can save your search.
+
+1. Put in the search parameters, such as applications or services, add filters, and click <strong>Search</strong>.
+1. Click the bookmark icon, specify a name for your search, and click <strong>Save</strong>.
+   ![Bookmark icon brings up save search dialog](/images/save_a_search_button.png)
+1. Click the <strong>Lists</strong> button to the left of the query.
+   ![List button](/images/saved_searches_list_button.png)
+1. In the dialog that appears: 
+    <ul>
+      <li>Click <strong>My Recent Searches</strong> for a list of your recent searches. </li>
+      <li>Click <strong>My Saved Searches</strong> for a complete list of your searches. </li>
+      <li>Click <strong>All Saved Searches</strong> for a list of searches in your organization.  </li>
+    </ul>
+    ![List button](/images/saved_searches_list.png)
+    
+1. Click a saved search to select it.
+
+### Delete Saved Searches
+
+The Lists button gives you access to your own saved searches and to saved searches from others in your organization.
+
+To delete a saved search from your list:
+<table style="width: 100%;">
+  <tr>
+    <td width="50%">
+      <ol><li>Select the list.</li>
+      <li>Click <strong>Manage</strong></li>
+      <li>Select <strong>Delete</strong> from the ellipsis icon to the left of that saved search.</li>
+      <li>Click <strong>Delete</strong> and confirm.</li>
+      </ol>
+      </td>
+      <td markdown="span" width="50%">
+        ![Elipsis icon and delete option](/images/delete_saved_search.png)
+      </td>
+  </tr>
+</table>
+
+### Set A Saved Search as the Default Search
+
+You can manage your saved searches to set a saved search as your default search.
+
+User's with the **Applications** permission can select a saved search as an Organization Default and set a saved search as the default search for all users. If you want to update the default search result, you need to save a new search and follow the steps below to make it your default search.
+
+1. Click the <strong>Lists</strong> button to the left of the query.
+    ![List button](/images/saved_searches_list_button.png)
+1. Click **Manage**.
+1. Set the default search:
+    * If you are a user with **Applications** permissions, you can set a default search for all users. Click the drop-down under the **Set Default** column, and select **Organization Default**.
```

Comment:
If you are a user with the **Applications** permission, you can set a default search for all users. Click the drop-down under the **Set Default** column, and select **Organization Default**.